3party/vulkan_wrapper/CMakeLists.txt

```diff
@@ -17,9 +17,11 @@ endif()
 
 target_include_directories(${PROJECT_NAME} PUBLIC .)
 
-if (PLATFORM_LINUX)
-  target_compile_definitions(${PROJECT_NAME} PRIVATE VK_USE_PLATFORM_XLIB_KHR)
-endif()
+target_compile_definitions(${PROJECT_NAME}
+  PRIVATE
```

Comment:
Should it be PUBLIC, so any other library that directly or indirectly is using vulkan_wrapper won't be required to copy-pasting these definitions?